### PR TITLE
Added Binary Tree Preorder and Postorder Traversal Code

### LINKED ISSUE #723: 0144 - Binary Tree Preorder Traversal
## Description of the Problem
Given the root of a binary tree, return the preorder traversal of its nodes' values.

 

Example 1:

1 - 2 - 3

Input: root = [1,null,2,3]
Output: [1,2,3]

Example 2:
Input: root = []
Output: []

Example 3:
Input: root = [1]
Output: [1]

Example 4:

2 - 1

Input: root = [1,2]
Output: [1,2]

Example 5:

1 - 2

Input: root = [1,null,2]
Output: [1,2]

 

Constraints:

    The number of nodes in the tree is in the range [0, 100].
    -100 <= Node.val <= 100

## Code
```
# Definition for a binary tree node.
# class TreeNode(object):
#     def __init__(self, val=0, left=None, right=None):
#         self.val = val
#         self.left = left
#         self.right = right
class Solution(object):
    def preorderTraversal(self, root):
        """
        :type root: TreeNode
        :rtype: List[int]
        """
        
```

## Link To The LeetCode Problem
[LeetCode](https://leetcode.com/problems/binary-tree-preorder-traversal/)

### LINKED ISSUE #724: 0145 - Binary Tree Postorder Traversal
## Description of the Problem
Given the root of a binary tree, return the postorder traversal of its nodes' values.

 

Example 1:

1 - 2 - 3

Input: root = [1,null,2,3]
Output: [3,2,1]

Example 2:

Input: root = []
Output: []

Example 3:

Input: root = [1]
Output: [1]

Example 4:

2 - 1
 
Input: root = [1,2]
Output: [2,1]

Example 5:

1 - 2

Input: root = [1,null,2]
Output: [2,1]

 

Constraints:

    The number of the nodes in the tree is in the range [0, 100].
    -100 <= Node.val <= 100

 

Follow up:

Recursive solution is trivial, could you do it iteratively?

## Code
```
# Definition for a binary tree node.
# class TreeNode(object):
#     def __init__(self, val=0, left=None, right=None):
#         self.val = val
#         self.left = left
#         self.right = right
class Solution(object):
    def postorderTraversal(self, root):
        """
        :type root: TreeNode
        :rtype: List[int]
        """
        
```

## Link To The LeetCode Problem
[LeetCode](https://leetcode.com/problems/binary-tree-postorder-traversal/)

### PR DESCRIPTION
**Pull request by RgnDunes**
**IMPORTANT: Please do not create a Pull Request without an issue.**

### All Submissions:

* [x] I have read the CONTRIBUTING document.
* [x] My code is written in **Python3** and is ending with _.py_
* [x] Have you checked to ensure there aren't other open [Pull Requests](../../../pulls) for the same update/change?
* [x] I have checked that my submission does pass the test on LeetCode.com 
* [x] Does your filename follow the naming Conventions?
* [x] Have you linked your PR to an Issue?

**Closing issues**

closes #723

### CODE REVIEW

**Review by riwim — CHANGES_REQUESTED** [review]
Thanks for your contribution ✨

Please
1. Read CONTRIBUTING.md
1. Submit **only one solution**
1. Do not change `PULL_REQUEST_TEMPLATE.md`
1. Remove backticks around `closes #XXXX` in your comment (to properly link your opened issue)

**Comment by RgnDunes** [comment]
> Thanks for your contribution ✨
> 
> Please
> 
> 1. Read CONTRIBUTING.md
> 2. Submit **only one solution**
> 3. Do not change `PULL_REQUEST_TEMPLATE.md`
> 4. Remove backticks around `closes #XXXX` in your comment (to properly link your opened issue)

Made all the necessary changes you asked for.
> 1. Read
> 2. Deleted one solution so adding just one solution.
> 3. Undone all the changes made in `PULL_REQUEST_TEMPLATE.md`
> 4. Removed backticks around `closes #XXXX` in my comment (to properly link your opened issue)

File: PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -2,12 +2,12 @@
...
-* [ ] Have you checked to ensure there aren't other open [Pull Requests](../../../pulls) for the same update/change?
-* [ ] I have checked that my submission does pass the test on LeetCode.com 
-* [ ] Does your filename follow the naming Conventions?
-* [ ] Have you linked your PR to an Issue?
+- [ ] I have read the CONTRIBUTING document.
+- [ ] My code is written in **Python3** and is ending with _.py_
+- [ ] Have you checked to ensure there aren't other open [Pull Requests](../../../pulls) for the same update/change?
+- [ ] I have checked that my submission does pass the test on LeetCode.com
+- [ ] Does your filename follow the naming Conventions?
+- [ ] Have you linked your PR to an Issue?
```
Comment (riwim) [inline PULL_REQUEST_TEMPLATE.md]: 1., 2. and 4. seems good.
You still didn't reset the template to its original state.
```suggestion
* [ ] I have read the CONTRIBUTING document.
* [ ] My code is written in **Python3** and is ending with _.py_
* [ ] Have you checked to ensure there aren't other open [Pull Requests](../../../pulls) for the same update/change?
* [ ] I have checked that my submission does pass the test on LeetCode.com 
* [ ] Does your filename follow the naming Conventions?
* [ ] Have you linked your PR to an Issue?
```


Copy&paste the original content or (my recommendation) read about the several possibilities to undo changes in Git:
[Undoing Things](https://git-scm.com/book/en/v2/Git-Basics-Undoing-Things)

**Review by riwim — CHANGES_REQUESTED** [review]
I put some suggested changed in a comment.

File: PULL_REQUEST_TEMPLATE.md (hunk 1 above)
Comment (RgnDunes) [reply PULL_REQUEST_TEMPLATE.md]: Sir I have copied and pasted original content from original PULL_REQUEST_TEMPLATE.md file and commited it.

**Comment by riwim** [comment]
Good work 🎉

**Review by vJechsmayr — APPROVED** [review]
Thanks for contributing - Merged!

Runtime: 28 ms

Happy Hacktoberfest!

**Pull request merged**